src/styles/chefSection.scss

```diff
@@ -9,12 +9,21 @@
   width: 1185px;
   margin: 30px auto;
   gap: 40px;
+
+  @include mobile {
+    width: 100vw;
```

Comment:
I think using width: 100%; is a better practice since 100vw width includes the vertical scroll width, and can sometimes create unwanted horizontal scrolling